ql/src/test/queries/clientpositive/prepare_plan.q

```diff
@@ -0,0 +1,49 @@
+--! qt:dataset:src
+--! qt:dataset:alltypesorc
+
+set hive.explain.user=false;
+--explain select * from src where key > '0' limit 10;
+set hive.vectorized.execution.enabled=false;
+
+
+explain extended prepare pcount as select count(*) from src where key > $1;
+prepare pcount as select count(*) from src where key > $1;
+execute pcount (1,200);
+select count(*) from src where key > '200';
+
+-- single param
+explain extended prepare p1 as select * from src where key > $1 order by key limit 10;
+explain extended select * from src where key > '200' order by key limit 10;
+prepare p1 as select * from src where key > $1 order by key limit 10;
+
+execute p1 (1,200);
+select * from src where key > '200' order by key limit 10;
```

Comment:
I don't think it is necessary to execute the same query twice if it is just to show that the results are the same. If the results are verified when .out is committed I think it's sufficient.